src/Result/WebhookAuthorizedEventList.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+class WebhookAuthorizedEventList extends AbstractListResult
```

Comment:
I think this makes no sense here. The returned data of `authorizedEvents` from the getWebhook is an object and no array https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Webhokks_GetWebhook

So I think you can remove this and rename class `WebhookAuthorizedEvent` to `WebhookAuthorizedEvents` and be good.

Reply:
You're right, made the change.